packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -67,6 +67,16 @@ export function getManualInspectorModeAttributes(
     manuallyTagged: true,
   };
 
+  if (!sharedProps.fieldId) {
+    debug.warn('Tagged element is missing field ID attribute and cannot be tagged', {
```

Comment:
This reads a bit weird, as in how can a tagged element not be tagged 😅

Reply:
That's very true 😅 I'll try to think of a different wording

Reply:
I guess you could also say something like "Element is missing field ID attribute so it cannot be tagged"